### DISCUSSION TITLE
Can a CKAN communicate with another CKAN?

### DISCUSSION THREAD

**Discussion by SofiaMiracle** [ckan/ckan#7774] [Q&A] [answered]
I have been searching the internet for a plugin (or something of the sort) that can make a connection between two separate CKAN's. So that if I for example create a dataset on one CKAN, it can then also be accessed on a separate CKAN. 
Is there a plugin for this, or has it been done before? If not; is it even possible? 

Edit: Something similar to Uncanny Automator for Wordpress (?). Maybe?

Thank you for reading :D

**Comment by amercader** [comment, marked as answer]
Hi @SofiaMiracle , you can definitely synchronize two CKAN instances by using the API in some capacity. You can set up a [harvester](https://github.com/ckan/ckanext-harvest) in instance B that regularly imports datasets from instance A, or use [ckanapi](https://github.com/ckan/ckanapi/blob/master/README.md#bulk-dumping-and-loading)'s bulk export / import to achieve a similar result, and with support for other entities like orgs, users, etc

**Reply by SofiaMiracle** [reply]
Thank you. This is very helpful.
